src/main/java/ymfc/commands/FindCommand.java

```diff
@@ -50,6 +55,7 @@ public void execute(RecipeList recipes, IngredientList ingredientList, Ui ui, St
                         isFoundInSteps(recipe)
                 )
                 .collect(Collectors.toCollection(ArrayList::new));
+        numMatches = results.size();
         ui.printFind(results, results.size());
```

Comment:
Can you replace `results.size()` here at line 58 to `numMatches` if you decided to introduce a new variable?

Reply:
Aight, will do